agent/lib/kontena/actors/container_exec.rb

```diff
@@ -27,30 +27,80 @@ def input(input)
       end
     end
 
+    # @param [Hash] size
+    # @option size [Integer] width
+    # @option size [Integer] height
+    def tty_resize(size)
+      return unless @container_exec
+
+      begin
+        @container_exec.resize({
+          'w' => size['width'], 'h' => size['height']
+        })
+      rescue Docker::Error::NotFoundError
+        sleep 0.1
+        retry if @container_exec
```

Comment:
Still don't understand the retry. The container exec has an ID, and once Docker is done with that exec, and the `POST /exec/:id/resize` starts returning HTTP 404 errors, how is retrying going to help?